extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/security/token/OneTimeAuthenticationTokenSender.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.vertx.http.security.token;
+
+import io.quarkus.security.credential.PasswordCredential;
+import io.quarkus.security.identity.SecurityIdentity;
+import io.smallrye.mutiny.Uni;
+
+/**
+ * An interface responsible for delivering of the one-time authentication token to the user.
+ * This interface should be implemented as {@link jakarta.enterprise.context.ApplicationScoped}
+ * or {@link jakarta.inject.Singleton} CDI bean.
+ */
+public interface OneTimeAuthenticationTokenSender {
```

Comment:
Would it be correct to name it `FormAuthenticationTokenSender`, with the one-time aspect highlighted in the docs ? Form authentication is the only built in mechanism that this interface can work with